torch/csrc/api/include/torch/nn/functional/linear.h

```diff
@@ -12,6 +12,12 @@ inline Tensor bilinear(const Tensor& input1, const Tensor& input2, const Tensor&
 
 // ============================================================================
 
+inline Tensor bias(const Tensor& input, const Tensor& bias) {
+  return input + bias;
```

Comment:
Obviously for this op the implementation is fairly trivial, but we generally have the C++ functional forms call the underlying ops to avoid duplication of logic. The core operator logic is then defined in only a single place.